### DISCUSSION TITLE
How do you know which firmware for which router?

### DISCUSSION THREAD

**Discussion by Dweinbach** [gnuton/asuswrt-merlin.ng#570] [General]
I have an XT8 and an ET8. I can't find anywhere that maps the model number to the firmware version. I was able to find it a few months ago, but now I can't. Could this be added to the readme?

**Comment by gnuton** [comment]
isn't this map in the main page?
![image](https://github.com/gnuton/asuswrt-merlin.ng/assets/576580/cc9f4597-33bb-417f-825f-715c7767cdc4)
